VirtoCommerce.Platform.Web/appsettings.json

```diff
@@ -18,6 +18,7 @@
     },
 
     "EmailSendingOptions": {
```

Comment:
We should define module specific settings within individual configuration section for each module. 
"Notification" for  Notification module